docs/server/source/appendices/run-with-ansible.md

```diff
@@ -26,36 +26,113 @@ Navigate to `bigchaindb/pkg/scripts` and run the `bootstrap.sh` script to instal
 
-OR
+Edit `bdb-config.yml` configuration file as per your requirements, sample configuration file(s):
+```text
+---
+deploy_docker: false #[true, false]
+docker_replset_size: 1 # Only needed if `deploy_docker` is true
+bdb_hosts:
+  - name: "<LOCAL_DEV_HOST_HOSTNAME>"
+```
+**Note**: You can also orchestrate a multi-node BigchainDB cluster on a local dev host using Docker containers.
+Here is a sample `bdb-config.yml`
+```text
+---
+deploy_docker: true #[true, false]
+docker_replset_size: 3
+bdb_hosts:
+  - name: "<LOCAL_DEV_HOST_HOSTNAME>"
+```
+
+#### Remote Setup | Ansible
+You can also run the Ansible playbook `bdb-deploy.yml` on remote machine(s) and set up the BigchainDB node where
+BigchainDB can be run as a process or inside a Docker container(s) depending on your configuration.
+
+Before, running the playbook on a remote host, you need to update the `hosts` and `bdb-config.yml` configuration, which will notify Ansible that we need to run the play on a remote host.
+
+##### Update Hosts | Remote
+Navigate to `bigchaindb/pkg/configuration/hosts` inside the BigchainDB repository.
+```text
+$ cd bigchaindb/pkg/configuration/hosts
+```
+
+Edit `all` configuration file:
+```text
+# Delete any existing configuration in this file and insert
+<Remote_Host_IP/Hostname> ansible_ssh_user=<USERNAME> ansible_sudo_pass=<ROOT_PASSWORD>
+```
+
+**Note 1**: You can multiple hosts to `all` configuration file. Root password is needed because ansible
+will run some tasks that require root permissions.
+
+**Note 2**: You can also use other methods to get inside the remote machines instead of password based SSH. For other methods
+please consult [Ansible Documentation](http://docs.ansible.com/ansible/latest/intro_getting_started.html).
+
+##### Update Configuration | Remote
+Navigate to `bigchaindb/pkg/configuration/vars` inside the BigchainDB repository.
+```text
+$ cd bigchaindb/pkg/configuration/vars/bdb-config.yml
+```
+
+Edit `bdb-config.yml` configuration file as per your requirements, sample configuration file(s):
+```text
+---
+deploy_docker: false #[true, false]
+docker_replset_size: 1 # Only needed if `deploy_docker` is true
+bdb_hosts:
+  - name: "<REMOTE_MACHINE_HOSTNAME>"
+```
+
+### BigchainDB Setup | Ansible
+Now, You can safely run the `bdb-deploy.yml` playbook and everything will be taken care of by `Ansible`. To run the playbook please navigate to the `bigchaindb/pkg/configuration` directory inside the BigchainDB repository and run the `bdb-deploy.yml` playbook.
+
+```text
+$ cd bigchaindb/pkg/configuration/
 
-# To deploy all services inside docker containers
-$ sudo ansible-playbook quickstart.yml --extra-vars "with_docker=true" -c local
+$ sudo ansible-playbook bdb-deploy.yml -i /bigchaindb/configuration/hosts/all
 ```
 
-After successfull execution of the playbook, you can verify that BigchainDB docker/process is running.
+After successfull execution of the playbook, you can verify that BigchainDB docker(s)/process(es) is(are) running.
```

Comment:
i tried sending transactions on `localhost:<bdb_container_por>`, `host_ip:<bdb_container_port>` and `docker0_bridge_ip:<bdb_container_port>` but all seem to fail with connection refused for some reason. also I see a lot of stale bigchaindb processes running on my machine and multiple bigchaindb processes running inside each container as well. we should add some cleanup workflows

Reply:
I will look into it and get back to you.